### ISSUE TITLE
Cannot parse xml grammar

### ISSUE THREAD

**Issue by heypnus** [CodeFalling/hexo-renderer-org#38]
When I insert xml code as below:

```
#+BEGIN_SRC xml
   <Command>
     <Order>1</Order>
   </Command>
#+END_SRC
```

Hexo gives me:

```
&lt;Command&gt;
  &lt;Order&gt;1&lt;/Order&gt;
&lt;/Command&gt;
```

**Comment by CodeFalling** [comment]
fixed in 6b815f2, try upguard `hexo-renderer-org`.

**Comment by heypnus** [comment]
OK. After upgrade, need to run **`hexo clean`** before `hexo s`.

Thanks a lot 👍
